crates/zksync/core/src/lib.rs

```diff
@@ -240,3 +241,59 @@ pub fn to_safe_address(address: Address) -> Address {
         address
     }
 }
+
+const SIGNATURE_CREATE: &str = "create(bytes32,bytes32,bytes)";
```

Comment:
Can we document the original signature? From `ContractDeployer`, `create2(...)` etc